pro1/pro1/settings.py

```diff
@@ -136,9 +137,9 @@
 LOGIN_URL = 'login'
 LOGOUT_URL = 'logout'
 
-EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_PORT = 587
-EMAIL_USE_TLS = True
+EMAIL_HOST = config('EMAIL_HOST')
+EMAIL_PORT = config('EMAIL_PORT', cast=int)
+EMAIL_USE_TLS = config('EMAIL_USE_TLS', cast=bool)
 
 EMAIL_HOST_USER = os.getenv('EMAIL_USER')
```

Comment:
@akashgiricse : If possible, set up something so that we can store email id and password in an encrypted form. Take your time, no hurry. Nice job btw 💯

Reply:
Hey @monsij , I could not find a useful solution for storing the password in encrypted form. I can write a python script to use Django's inbuilt `hashers` but eventually one has to configure his own mail *EMAIL_HOST_USER* and *EMAIL_HOST_PASSWORD* to work on emailing services on their own end. This is what I could research about. There may be a solution but I don't know.